controllers/servicediscovery/servicediscovery_controller.go

```diff
@@ -246,6 +246,7 @@ func newLighthouseAgent(cr *submarinerv1alpha1.ServiceDiscovery, name string) *a
 								{Name: "SUBMARINER_CLUSTERID", Value: cr.Spec.ClusterID},
 								{Name: "SUBMARINER_DEBUG", Value: strconv.FormatBool(cr.Spec.Debug)},
 								{Name: "SUBMARINER_GLOBALNET_ENABLED", Value: strconv.FormatBool(cr.Spec.GlobalnetEnabled)},
+								{Name: "SUBMARINER_HALTONCERTIFICATEERROR", Value: strconv.FormatBool(cr.Spec.HaltOnCertificateError)},
```

Comment:
This looks mouthful, what about the following?

```suggestion
								{Name: "SUBMARINER_HALT_ON_CERT_ERROR", Value: strconv.FormatBool(cr.Spec.HaltOnCertificateError)},
```

Reply:
Fixed in all three PRs, thanks for the suggestion!